config/ssh.yaml

```diff
@@ -0,0 +1,19 @@
+# ssh configuration
+
+# cluster_id: <id of the cluster to connect to>
+cluster_id: spark_cluster
```

Comment:
Now that I'm looking at this, I'm not sure if we should be adding the cluster id to this file. The ssh should be valid for any ssh session to any cluster (and we should expect that users create and throw clusters away often) so adding the cluster id in here seems to go against that principle. 

Thoughts?

Reply:
adding cluster id has always been weird for me. for ssh and for create cluster. i think id should be mandatory param for both cluster creation and cluster ssh

for spark cluster create:
The scenario of someone getting an error because of duplicate cluster name is too likely. For simplicity, i think we should just make it a standard that id needs to be specified for each of the azb spark cluster xyz commands.

for azb spark cluster ssh - prefilling the cluster_id in the configs seems like not a good idea i think.. because how do we know that the user is gonna have a cluster called "spark_cluster"

Reply:
I agree that errors because of duplicate clusters is very likely with how things are currently set up. Moving id to a parameter seems like a fine solution.

The reason that the default ssh.yaml file has spark_cluster as the cluster_id is because the same id is used as default in cluster.yaml. This is so users could type `azb spark cluster create; azb spark cluster ssh` and it would just work.